src/backend/storeManagers/sideload/library.ts

```diff
@@ -27,7 +27,8 @@ export function addNewApp({
     title,
     install: {
       executable,
-      platform
+      platform,
+      is_dlc: false
```

Comment:
mark it as not DLC just to be explicit since now we check if `is_dlc` is true to ignore the shortcuts